contracts/jobs/JobsManager.sol

```diff
@@ -451,11 +449,22 @@ contract JobsManager is ManagerProxyTarget, IVerifiable, IJobsManager {
         external
         whenSystemNotPaused
         onlyVerifier
+        jobExists(_jobId)
     {
-        address transcoder = jobs[_jobId].transcoderAddress;
+        Job storage job = jobs[_jobId];
```

Comment:
Assuming that the verifier contract being used is the LivepeerVerifier, these checks should prevent a rogue whitelisted solver from triggering this function via `__callback` in LivepeerVerifier` to slash a transcoder. However, there aren't any checks preventing a rogue whitelisted solver from triggering this function via `__callback` with result set to true, but in that case the only thing that happens is the `PassedVerification` event is fired so I think that should be ok

Reply:
If the claim does not exist, `Claim storage claim = job.claims[_claimId]` should throw with an out of bounds error https://github.com/livepeer/protocol/pull/196/files#diff-8f91d5b27bb5716159a6dc1367d9dffbR572